Userland/Libraries/LibJS/Runtime/Temporal/AbstractOperations.cpp

```diff
@@ -421,20 +421,20 @@ ThrowCompletionOr<SecondsStringPrecision> to_seconds_string_precision(VM& vm, Ob
     // 15. If fractionalDigitCount is 1, 2, or 3, then
     if (fractional_digit_count == 1 || fractional_digit_count == 2 || fractional_digit_count == 3) {
         // a. Return the Record { [[Precision]]: fractionalDigitCount, [[Unit]]: "millisecond", [[Increment]]: 10^(3 - fractionalDigitCount) }.
-        return SecondsStringPrecision { .precision = fractional_digit_count, .unit = "millisecond"sv, .increment = (u32)pow(10, 3 - fractional_digit_count) };
+        return SecondsStringPrecision { .precision = fractional_digit_count, .unit = "millisecond"sv, .increment = (u32)pow(10., 3 - fractional_digit_count) };
```

Comment:
Why is this necessary? Won't the compiler cast this the int literal to a double anyway?

Reply:
Whoops just left from trying some stuff